### PR TITLE
[CDAP-7199] fix incorrect documentation for updating table schema

### PR DESCRIPTION
**Pull request by anew**
the docs had an incorrect JSON in the request body. Replaced with correct (and tested) body.

### CODE REVIEW

**Comment by albertshau** [comment]
lgtm

File: cdap-docs/developers-manual/source/data-exploration/tables.rst [hunk 1]
```diff
@@ -75,25 +75,25 @@ For example, the same schema set through the example code above can also be set
...
+          }", \
+          "schema.row.field": "id" \
         }'
   
 CDAP schemas are adopted from the `Avro Schema Declaration <http://avro.apache.org/docs/1.7.3/spec.html#schemas>`__.
-Note that since dataset properties must be strings, the schema JSON has to be escaped properly.
+Note that since dataset properties must be strings, the schema JSON has to be escaped properly. Note also that
+the properties given in this request replace the existing properties entirely; that is, if you had set other
+properties for this table, such as time-to-live (``dataset.table.ttl``), you must also include these properties in the
+update request.
```
Comment (awholegunch) [inline cdap-docs/developers-manual/source/data-exploration/tables.rst]: What about using:

```
Note also that the properties given in this request replace all existing properties; 
that is, if you have set other properties for this table, such as time-to-live 
(``dataset.table.ttl``), you must also include values for those properties in the update request.
```

**Comment by awholegunch** [comment]
Minor change to wording.

**Comment by awholegunch** [comment]
Are there any other uses of this in the documentation, such as the RESTful API?
(checking...)

**Comment by awholegunch** [comment]
On http://docs.cask.co/cdap/current/en/reference-manual/http-restful-api/dataset.html#updating-an-existing-dataset we don't explain about how this replaces _all_ existing properties. I would think it should be added there.

**Comment by awholegunch** [comment]
In fact we say the opposite:
`or as a basis for updating some properties of this dataset without modifying the remaining properties.`

**Comment by awholegunch** [comment]
Have seen that, I'd say this in NLGTM.

File: cdap-docs/developers-manual/source/data-exploration/tables.rst (hunk 1 above)
Comment (awholegunch) [reply cdap-docs/developers-manual/source/data-exploration/tables.rst]: Why would the API be this way (replacing all)? Are you certain about this?

**Comment by awholegunch** [comment]
Let me add a change to this PR.

**Comment by awholegunch** [comment]
I've made an addition and a change. Please take a look. LGTM.

File: cdap-docs/developers-manual/source/data-exploration/tables.rst (hunk 1 above)
Comment (anew) [reply cdap-docs/developers-manual/source/data-exploration/tables.rst]: yes.

**Comment by anew** [comment]
@awholegunch your changes look ok to me. Is this LGTM then?

**Comment by awholegunch** [comment]
It's LGTUs. (LGTM x 2)

**Comment by anew** [comment]
there is already LGTM. Squashing and merging

**Pull request merged**